UnityMirror/UnityServer/UserSettings/Layouts/default-2021.dwlt

```diff
@@ -1,30 +1,6 @@
 %YAML 1.1
 %TAG !u! tag:unity3d.com,2011:
 --- !u!114 &1
-MonoBehaviour:
```

Comment:
Why would these values change here? The change of this PR is only about reading / writing port number if I am not wrong.

Reply:
Unity UI editor did these changes, not sure.